balkhash/__init__.py

```diff
@@ -1,8 +1,17 @@
 from balkhash.leveldb import LevelDBDataset
 from balkhash.datastore import GoogleDatastoreDataset
+from balkhash.postgres import PostgresDataset
 
+BACKENDS = {
```

Comment:
We need to find another way of doing this, but let's not make it subject to this PR. I feel we might want to segment dependencies, too, and make the imports conditional.